packages/zoe/src/typeGuards.js

```diff
@@ -79,3 +121,231 @@ export const InvitationElementShape = M.splitRecord({
 export const OfferHandlerI = M.interface('OfferHandler', {
   handle: M.call(SeatShape).optional(M.any()).returns(M.string()),
 });
+
+export const SeatHandleAllocations = M.arrayOf(
+  harden({
+    seatHandle: SeatShape,
+    allocation: AmountKeywordRecordShape,
+  }),
+);
+
+export const ZoeMintI = M.interface('ZoeMint', {
+  getIssuerRecord: M.call().returns(IssuerRecordShape),
+  mintAndEscrow: M.call(AmountShape).returns(),
+  withdrawAndBurn: M.call(AmountShape).returns(),
+});
+
+export const ZcfMintI = M.interface('ZcfMint', {
+  getIssuerRecord: M.call().returns(IssuerRecordShape),
+  mintGains: M.call(AmountKeywordRecordShape, M.remotable('zcfSeat')).returns(),
+  burnLosses: M.call(
+    AmountKeywordRecordShape,
+    M.remotable('zcfSeat'),
+  ).returns(),
+});
+
+export const ExitObjectI = M.interface('Exit Object', {
+  exit: M.call().returns(),
+});
+
+export const InstanceAdminShape = M.remotable('InstanceAdmin');
+export const InstanceAdminI = M.interface('InstanceAdmin', {
+  makeInvitation: M.call(InvitationHandleShape, M.string())
+    .optional(M.any(), M.pattern())
+    .returns(M.promise()),
+  saveIssuer: M.callWhen(M.await(IssuerShape), M.string()).returns(
+    IssuerRecordShape,
+  ),
+  isBlocked: M.call(M.string()).returns(M.boolean()),
+  makeZoeMint: M.call(KeywordShape)
+    .optional(AssetKindShape, DisplayInfoShape, M.pattern())
+    .returns(M.remotable('zoeMint')),
+  registerFeeMint: M.call(KeywordShape, M.remotable('feeMintAccess')).returns(
+    M.remotable('feeMint'),
+  ),
+  makeNoEscrowSeat: M.call(
+    AmountKeywordRecordShape,
+    ProposalShape,
+    M.remotable('ExitObj'),
+    SeatShape,
+  ).returns({ userSeat: SeatShape }),
```

Comment:
If this only returns the one thing, could it just return it, rather than a singleton kit containing it? Is it too late because compat?

```suggestion
  ).returns(SeatShape),
```

Reply:
Not too late.

Reply:
Whew! Looks done. Resolving.